### DISCUSSION TITLE
Choose directories for routes, markers and notes

### DISCUSSION THREAD

**Discussion by trymeouteh** [osmandapp/OsmAnd#20158] [Ideas]
Please add a setting to choose what directory the app will store user created routes, markers and notes. I would like to change them personally to _Documents/Maps/Routes/_ and _Documents/Maps/Markers/_ myself, but to be able to allow users to choose the directory will be best.

**Comment by DmitryAlexei** [comment]
Please see docs
https://osmand.net/docs/user/troubleshooting/maps-data#picking-a-generally-accessible-osmand-data-storage-folder-using-the-media-storage
